service/src/main/java/org/sunbird/actor/user/validator/UserRequestValidator.java

```diff
@@ -798,29 +798,30 @@ public String validateUserType(
       if (StringUtils.isBlank(stateCode)) {
         stateCode = JsonKey.DEFAULT_PERSONA;
       }
-
-      if (!userTypeConfigMap.containsKey(stateCode)) {
-        // Get profile data config
-        Map<String, List<String>> userProfileConfigMap =
-            FormApiUtil.getUserTypeConfig(FormApiUtil.getProfileConfig(stateCode, context));
-        if (MapUtils.isEmpty(userProfileConfigMap)) {
-          // Get Default Config
-          stateCode = JsonKey.DEFAULT_PERSONA;
-          userProfileConfigMap = userTypeConfigMap.get(stateCode);
+      if(Boolean.getBoolean(System.getenv(JsonKey.IS_FORM_VALIDATION_REQUIRED))) {
```

Comment:
@BharathwajShankar Only handling this here may not work out. Because FormApiUtil is referenced from other classes also. So put a mock method in FormApiUtil or handler itself instead of checkin this in multiple places.

Reply:
have handled as discussed. hence resolving